execution/interface.go

```diff
@@ -56,7 +56,7 @@ type ExecutionSequencer interface {
 	ForwardTo(url string) error
 	SequenceDelayedMessage(message *arbostypes.L1IncomingMessage, delayedSeqNum uint64) error
 	NextDelayedMessageNumber() (uint64, error)
-	GetL1GasPriceEstimate() (uint64, error)
+	TrimCache(to arbutil.MessageIndex)
```

Comment:
The name isn't great. This is consensus telling execution how many messages are we got from Parent chain, or saying where backlog begins.. or something like that.
Name something like MarkPosInParentChain? MarkFeedStart? BatchReceieved?
something like that..

Reply:
Renamed to MarkFeedStart.